src/spatialref.jl

```diff
@@ -616,6 +618,31 @@ function toPROJ4(spref::AbstractSpatialRef)::String
     return unsafe_string(projptr[])
 end
 
+"""
+    toEPSG(spref::AbstractSpatialRef)
+
+If possible, export EPSG code for this coordinate system.
+"""
+function toEPSG(spref::AbstractSpatialRef)::String
+    result = GDAL.osrgetauthorityname(spref.ptr, "PROJCS")
+    projcs = "PROJCS"
+
+    if isnothing(result)
+        result = GDAL.osrgetauthorityname(spref.ptr, "GEOGCS")
+        projcs = "GEOGCS"
+    end
+
+    if isnothing(result)
+        error("No PROJCS or GEOGCS Authority found")
```

Comment:
Would people prefer an empty string, or an error in this case?

Reply:
I was thinking the same thing but passing an empty string or "EPSG:" to GFT.EPSG() will throw an error

Reply:
`GFT.EPSG` actually holds an integer internally, so the constructor checks that a string is `epsg:NNNN` and parses then Ns.

Reply:
so I think our only options are [1] throw an error or [2] set epsg = 0

Reply:
I'm okay with keeping the current/proposed behavior of raising an error, so I'll resolve this -- but feel free to re-open or create a ticket for discussion if it remains controversial.